docs/fundamentals/jcloud/advanced.md

```diff
@@ -43,7 +42,40 @@ executors:
     jcloud:
       capacity: on-demand
 ```
-(jcloud-external-executors)=
+
+## Storage
+
+We currently support 2 kind of storage type `ebs` and `efs`. The former one is block device and the second one is shared file system.
+
+ By default, we attach an `efs` to all the Executors in a Flow. The benefits of doing so are
+
+- It can grow in size dynamically, so you don't need to shrink/grow volumes as & when necessary.  
+- All Executors in the Flow can share a disk. 
+- The same disk can also be shared with another Flow by passing `--workspace-id <prev-flow-id>` while deploying a Flow.
+
+If your Executor needs high IO, you can use `ebs` instead. 
+- The disk cannot be shared with other Executors / Flows. 
+- You must pass a size of storage  (default: `1G`,  max `10G`)
```

Comment:
```suggestion
```{note}
By default, we attach an `efs` to all the Executors in a Flow. The benefits of doing so are

- It can grow in size dynamically, so you don't need to shrink/grow volumes as & when necessary.  
- All Executors in the Flow can share a disk. 
- The same disk can also be shared with another Flow by passing `--workspace-id <prev-flow-id>` while deploying a Flow.

If your Executor needs high IO, you can use `ebs` instead. 
- The disk cannot be shared with other Executors / Flows. 
- You must pass a size of storage  (default: `1G`,  max `10G`)
```
```

Please put this as a [note](https://pradyunsg.me/furo/reference/admonitions/#note).